src/Symfony/Component/Workflow/Tests/Dumper/GraphvizDumperTest.php

```diff
@@ -63,12 +63,14 @@ public function provideComplexWorkflowDefinition()
 
         $builder->addPlaces(range('a', 'g'));
 
-        $builder->addTransition(new Transition('t1', 'a', array('b', 'c')));
-        $builder->addTransition(new Transition('t2', array('b', 'c'), 'd'));
-        $builder->addTransition(new Transition('t3', 'd', 'e'));
-        $builder->addTransition(new Transition('t4', 'd', 'f'));
-        $builder->addTransition(new Transition('t5', 'e', 'g'));
-        $builder->addTransition(new Transition('t6', 'f', 'g'));
+        $builder->addTransitions(array(
+            array('t1', 'a', array('b', 'c')),
+            array('t2', array('b', 'c'), 'd'),
+            array('t3', 'd', 'e'),
+            array('t4', 'd', 'f'),
+            array('t5', 'e', 'g'),
+            array('t6', 'f', 'g'),
+        ));
```

Comment:
I originally removed the possibility to pass instances, and then I did not change back the tests because I found them much more readable like this, don't you?